BrainPortal/app/views/nh_invitations/_message_details.html.erb

```diff
@@ -0,0 +1,52 @@
+
+<%-
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2012
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.  
+#
+-%>
+<% messtype = mess.message_type
+   prefix = "flash"  # disguise an invite confirmation message for a flash notice until proper styling defined
+   messclass = "#{prefix}_#{messtype}"
+%>
+
+<% if mess.description.present? %>
+  <p class=<%= messclass %>>
```

Comment:
By the way, the values of the HTML attributes should be quoted. They were quoted in the original `messages/_message_details.html.erb`, why did you remove the quotes in your copy?

Reply:
I tried a bit improve over existing code.

Reply:
Note If I add quotes to this version I will get double quotes

Reply:
This is not fixed yet

Reply:
added